src/toast/util/toastHooks.ts

```diff
@@ -0,0 +1,13 @@
+import {useContext} from "react";
+
+import {ToastContext} from "../ToastProvider";
+
+function useToast() {
+  return useContext(ToastContext);
+}
+
+function useToaster() {
+  return useToast().dispatchToastAction;
+}
+
```

Comment:
I think adding a hook for directly showing toast would be great, because we mostly use "DISPLAY" action with autoClose feature, and for each display action we have to pass `type` and `payload`, this creates too much duplicated code IMO

```suggestion
function useDisplayToast() {
  const dispatchToast =  useToaster();
  
  function displayToast(payload:ToastData) {
    dispatchToast({
      type: "DISPLAY",
      payload
    });
  }
  
  return displayToast;
}
```

This would reduce:
```ts
dispatchToast({
  type: "DISPLAY",
  payload: {
    mode: "success",
    component: "Success Message"
  }
})
```
   to this:
```ts
displayToast({
   mode: "success",
   component: "Success Message"
})
```

This seems a little change but overall it can help reduce duplicated code.
Maybe we can discuss this in our weekly meeting